fenix/app/src/main/res/values/strings.xml

```diff
@@ -2301,6 +2301,10 @@
     <string name="translation_option_bottom_sheet_never_translate_in_language">Never translate %1$s</string>
     <!-- Toggle switch label that allows a user to set the setting if they would like the browser to never translate the site the user is currently visiting. -->
     <string name="translation_option_bottom_sheet_never_translate_site">Never translate this site</string>
+    <!-- Toggle switch description that will appear under the label if the switch is checked.If the switch label is "translation_settings_automatic_never_translate_sites" -->
```

Comment:
I'm not sure if pontoon can connect string names like `translation_settings_automatic_never_translate_sites` to the string, maybe substitute in the comment to be safe?

```suggestion
    <!-- Toggle switch description that will appear under the "Never translate these sites" settings toggle switch to provide more information on how this setting interacts with other settings. -->
```